helm/cluster/files/apps/external-dns.yaml

```
@@ -0,0 +1,20 @@
+appName: external-dns
+chartName: external-dns-app
+configKey: externalDns
+catalog: default
+clusterValues:
+  configMap: true
+  secret: true
+dependsOn: prometheus-operator-crd
+namespace: kube-system
+# used by renovate
+# repo: giantswarm/external-dns-app
+version: 3.1.0
+defaultValues:
+  txtOwnerId: giantswarm-io-external-dns
+  txtPrefix: {{ $.Values.global.metadata.name }}
```

we added this new value today which is needed for some cases. I'd be nice to include it here already.

```suggestion
  txtPrefix: {{ $.Values.global.metadata.name }}
  registry: txt
```

---

Do we need this for all providers?

I'm working ATM on enabling provider-specific config for apps, so every cluster-\<provider\> app can set provider-specific config it needs for that app (provider-specific config will be merged with and will overwrite provider-independent default values here).

---

yes, for all providers is fine. It's just a safeguard, because there is a value in the catalog overriding our default in the app.

---

Thanks, added it.